code/modules/mob/language.dm

```diff
@@ -55,7 +90,7 @@
 
 	var/input_size = length(input)
 	var/scrambled_text = ""
-	var/capitalize = 1
+	var/capitalize = 0
```

Comment:
```suggestion
	var/capitalize = FALSE
```
`capitalize` on lines 99 and 106 should also be changed to match our current coding conventions.